mpb/fields.c

```diff
@@ -706,32 +706,31 @@ number get_energy_point(vector3 p)
      return f_interp_val(p, mdata, (real *) curfield, 1, 0);
 }
 
-cvector3 get_bloch_field_point(vector3 p)
+void get_bloch_field_point_(scalar_complex *field, vector3 p)
```

Comment:
Maybe `scalar_complex field[3]` would be clearer here?

Reply:
I wasn't aware of this syntax but that does seem better in terms of readability.
Would it make sense to make it `scalar_complex field[static 3]` to guarantee the size exaplicitly to the compiler (that's what I understood from [trying to read up on this syntax](https://stackoverflow.com/a/26410652/9911781))?